src/js/components/keyword/table/ResultsTable.jsx

```diff
@@ -0,0 +1,137 @@
+/**
+ * ResultsTable.jsx
+ * Created by Lizzie Salita 1/8/18
+ **/
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import IBTable from 'components/sharedComponents/IBTable/IBTable';
+
+import ResultsTableHeaderCell from 'components/search/table/cells/ResultsTableHeaderCell';
+import ResultsTableFormattedCell from 'components/search/table/cells/ResultsTableFormattedCell';
+import ResultsTableAwardIdCell from 'components/search/table/cells/ResultsTableAwardIdCell';
+
+const propTypes = {
+    results: PropTypes.array,
+    columns: PropTypes.object,
+    visibleWidth: PropTypes.number,
+    loadNextPage: PropTypes.func,
+    currentType: PropTypes.string,
+    tableInstance: PropTypes.string,
+    sort: PropTypes.object,
+    updateSort: PropTypes.func
+};
+
+const rowHeight = 40;
+// setting the table height to a partial row prevents double bottom borders and also clearly
+// indicates when there's more data
+const tableHeight = 29.5 * rowHeight;
```

Comment:
Requesting 30 rows while displaying 29.5 is problematic because it immediately triggers a pagination request. The first page of the table being shown is actually the sometimes the second page (with the first page lost) due to weirdness with the delayed/batched nature of `setState` when the API responds very quickly.

You should either increase the the number of rows requested by 5 to 35 or decrease the number of rows shown by 4 to 25.5.

Reply:
Increased request to 35 rows